src/components/Identification/Components/LinksDialog/linksDialog.scss

```diff
@@ -1,4 +1,4 @@
-@import '../../../../../../vars';
+@import '../../../../vars';
 
```

Comment:
I moved the Identification component from inside the My Profile component to "src/components". Due to this, I had to change the file paths of imported files